Project.toml

```diff
@@ -20,6 +22,7 @@ ProgressLogging = "33c8b6b6-d38a-422a-b730-caa89a2f386c"
 RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
This shouldn't be added as a dependency.

Reply:
yeah, I believe it's fixed in the latest commit

Reply:
My bad, sorry